.github/workflows/ci.yaml

```
@@ -0,0 +1,108 @@
+      - name: Sleep for 2 seconds
+        run: sleep 2
+      - name: Dummy ESLint
+        run: echo "✅ ESLint Passed"
+
+  # iOS EXPERIMENTAL
+  ios-hold-build-experimental:
+    needs: eslint
+    runs-on: ubuntu-latest
+    environment: ios_hold_build_experimental
+    steps:
+      - run: echo "🛑 Hold: iOS Experimental Build Approved"
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 22: mapping values are not allowed in this context

(syntax-check)